src/app/routes/applications/applications.component.spec.ts

```diff
@@ -1,24 +1,78 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { MatDialog } from '@angular/material/dialog';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DialogService } from 'src/app/shared/services/dialog/dialog.service';
+import { EnvService } from 'src/app/shared/services/env/env.service';
+import { IamService } from 'src/app/shared/services/iam.service';
+import { UrlParamService } from 'src/app/shared/services/url-param.service';
+import { provideMockStore } from '@ngrx/store/testing';
+import { MockActivatedRoute, dialogSpy } from '@tests';
 import { ApplicationsComponent } from './applications.component';
+import { DomainsService } from 'iam-client-lib';
 
-xdescribe('ApplicationsComponent', () => {
+describe('ApplicationsComponent', () => {
   let component: ApplicationsComponent;
   let fixture: ComponentFixture<ApplicationsComponent>;
-
-  beforeEach(() => {
-    TestBed.configureTestingModule({
-      declarations: [ApplicationsComponent],
-    }).compileComponents();
-  });
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(ApplicationsComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+  let dialog;
+  let domainsService;
+  const routerSpy = jasmine.createSpyObj('Router', [
+    'navigateByUrl',
+    'navigate',
+  ]);
+  let activatedRouteStub: MockActivatedRoute;
```

Comment:
@dawidgil would love your thoughts on this approach. Maybe this is overkill to have tests at all for this, but I am trying to practice my Angular testing :) 
I am doing a before each before each test suite here because I need to override the ENS Service provider, and one solution I read about was to just compile it for each suite, because if you call compileComponents() at the top, the values cannot be overridden per describe.